src/Components/AlbumCard/AlbumCard.js

```diff
@@ -0,0 +1,23 @@
+import './AlbumCard.css'
+
+function AlbumCard({ coverImage, title, name, year, genres }) {
```

Comment:
Glad you are also a fan of destructuring props, this is consistent with what I have on my branch